src/Compilers/Core/Portable/Operations/IConditionalOperation.cs

```diff
@@ -31,6 +31,11 @@ public interface IConditionalOperation : IOperation
         /// Operation to be executed if the <see cref="Condition"/> is false.
         /// </summary>
         IOperation WhenFalse { get; }
+
+        /// <summary>
+        /// Is result a managed reference
+        /// </summary>
+        bool IsRef { get; }
```

Comment:
>bool IsRef { get; } [](start = 8, length = 19)

Expose RefKind property instead?

Reply:
> Expose RefKind property instead?

Because the kind of the reference is irrelevant here.